aikau/src/main/resources/alfresco/renderers/Thumbnail.js

```diff
@@ -160,6 +194,27 @@ define(["dojo/_base/declare",
       folderImage: "folder-64.png",
 
       /**
+       * Indicates whether or not the thumbnail image should be given a shadow effect.
+       *
+       * @instance
+       * @type {boolean}
+       * @default
+       * @since 1.0.40
+       */
+      hasShadow: false,
```

Comment:
This feels like CSS not a JS property. Can't we have a default that looks good and let people style this as necessary? Doesn't feel right having it as config, but I guess if there's a specific need to have it ... ?